dbt run-operation ref (#9193)
fishtown-analytics/docs.getdbt.com: website/docs/reference/commands/run-operation.md

The `dbt run-operation` command is used to invoke a macro or execute a SQL or Jinja string directly against the target database. For usage information on macro-based operations, refer to [operations](/docs/build/hooks-operations#about-operations).

<VersionBlock firstVersion="1.12">

> Macros are outside the group and access control system, so they are not subject to model [access](/reference/resource-configs/access) restrictions. This includes macros invoked using `dbt run-operation` &mdash; you can use `ref()` in a [macro](/docs/build/jinja-macros) to reference `private` or `protected` models regardless of their access level.

Reviewer: maybe we can add an example here?
```suggestion
Macros aren't subject to model [access](/reference/resource-configs/access) and [group](/reference/resource-configs/group) control system. As a result, a [macro](/docs/build/jinja-macros) can use `ref()` to reference `private` or `protected` models, including when the macro is invoked with `dbt run-operation`.

For example, a macro can reference a private model:

```jinja
{% macro example_macro() %}
  {{ ref('my_private_model') }}
{% endmacro %}
```
```

The `dbt run-operation` command is used to invoke a macro or execute a SQL or Jinja string directly against the target database. For usage information on macro-based operations, refer to [operations](/docs/build/hooks-operations#about-operations).

<VersionBlock firstVersion="1.12">

Macros aren't subject to model [access](/reference/resource-configs/access) and [group](/reference/resource-configs/group) control system. As a result, a [macro](/docs/build/jinja-macros) can use `ref()` to reference `private` or `protected` models, including when the macro is invoked with `dbt run-operation`.

> For example, a macro can reference a private model:

Reviewer 2: and then that macro can be run via `run-operation`
